tests/test_all_30_scenarios.py

```
@@ -0,0 +1,387 @@
+    def test_individual_scenario_complete(self, client, csrf_token, all_scenarios, scenario_number):
+        """各シナリオを個別に完全テスト（複数APIキーローテーション使用）"""
+        scenario_key = f"scenario{scenario_number}"
+        scenario = all_scenarios.get(scenario_key)
+        
+        if not scenario:
+            pytest.fail(f"シナリオ{scenario_number}が見つかりません")
+        
+        print(f"\n=== シナリオ{scenario_number}/30: {scenario.get('title', 'タイトル不明')} ===")
+        
+        # APIキー管理システムの状態確認
+        manager = get_api_key_manager()
```

Ruff (0.12.2)

Local variable `manager` is assigned to but never used

Remove assignment to unused variable `manager`

(F841)